.github/workflows/test-workflow.yml

```diff
@@ -8,10 +8,11 @@ on:
         type: string
 
 env:
-  RELAY_IMAGE_TAG: 0.32.0
-  NETWORK_NODE_IMAGE_TAG: 0.43.0
-  HAVEGED_IMAGE_TAG: 0.43.0
-  MIRROR_IMAGE_TAG: 0.89.0
+
+  RELAY_IMAGE_TAG: 0.36.0
+  NETWORK_NODE_IMAGE_TAG: 0.45.0-alpha.0
```

Comment:
Alpha1 is available if you want latest

Reply:
Yes, let's wait for https://github.com/hashgraph/hedera-mirror-node/pull/7327 to get in image, because the failing test in the CI here is because of what this fixes.